spec/dummy/db/schema.rb

```diff
@@ -261,9 +261,15 @@
 
   add_index "mc_payload_staged_instances", ["payload_staged_class_id"], name: "index_mc_payload_staged_instances_on_payload_staged_class_id", unique: true, using: :btree
 
+  create_table "mc_payload_stager_ancestor_handlers", force: true do |t|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.